.changeset/fast-singers-design.md

```diff
@@ -0,0 +1,16 @@
+---
+'@astrojs/vercel': minor
+---
+
+The entrypoints `@astrojs/vercel/serverless` and `@astrojs/vercel/static` are deprecated. Use the `@astrojs/vercel` entrypoint 
```

Comment:
```suggestion
Replaces the entrypoints `@astrojs/vercel/serverless` and `@astrojs/vercel/static` with a common `@astrojs/vercel` entrypoint.

You no longer specify an endpoint for your type of Vercel deployment, and you must update all your import statements to use the common entrypoint:
```
Would this be a correct way to say it? Or "Combines ... into"?

Reply:
Since this is a `minor`, I would like at least to tell the users that those entrypoints are *deprecated*, but they still work. I would like to avoid phrases like "replaced"/"removed" etc., because IMHO they hint a breaking change, which is not the case

Reply:
```suggestion
Deprecates the entrypoints `@astrojs/vercel/serverless` and `@astrojs/vercel/static`.  These will continue to work but are no longer documented and will be removed in a future version.  Update to the `@astrojs/vercel` entrypoint as soon as you are able:
```
How about something like this then?